auditlog/admin.py

```diff
@@ -26,5 +26,9 @@ def has_add_permission(self, request):
         # As audit admin doesn't allow log creation from admin
         return False
 
+    def get_queryset(self, request):
+        queryset = super().get_queryset(request).select_related("content_type", "actor")
```

Comment:
@2ykwang, why not use [`list_select_related`](https://docs.djangoproject.com/en/4.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_select_related)?

Reply:
Yes, @2ykwang Could you please give this a try and compare the numbers?